src/poly1305.rs

```diff
@@ -23,21 +23,15 @@ use core;
 
 // The assembly functions we call expect the state to be 8-byte aligned. We do
 // this manually, by copying the data into an aligned slice, rather than by
```

Comment:
The last line of the comment was cut off in an inlining mishap here, it looks like.